services/blockchain-connector/shared/sdk/constants/defaultResponses.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+const DEFAULT_RES_GET_STAKER = {
+	stakes: [],
+	pendingUnlocks: [],
+};
+
+const DEFAULT_RES_GET_CHAIN_ACCOUNT = {
+	lastCertificate: {},
+	name: null,
+	status: null,
+};
```

Comment:
Will these be reused somewhere? If not, then maybe we can define these in the respective files? Also, do we need a constant for the default structure? Usually, we just hardcode it in other cases. Ex: https://github.com/LiskHQ/lisk-service/blob/feature/1157-prepare-lisk-service-v070-for-beta/services/blockchain-indexer/shared/dataService/business/pos/stakers.js#L43-L51

Reply:
Currently not reused anywhere. Will apply the changes.

Reply:
Leaving it as is would've been fine because the fix from our end was most likely temporary. The changes will be rolled back once the SDK fixes it at their end. But let's leave it as is.